config/settings/local.py

```diff
@@ -104,3 +104,7 @@
 AUDIT_LOG_ENABLED = True
 
 DISABLE_RATELIMIT = True
+
+
+FILE_UPLOAD_BUCKET_ENDPOINT = "http://localstack:4566".format(FILE_UPLOAD_BUCKET)
```

Comment:
why use the format function if you are not using it?

Reply:
removed format